webapp/src/pages/app/projects/[projectName]/repos/[repoName]/list/[revision]/[[...path]]/index.tsx

```diff
@@ -169,10 +163,10 @@ cat ${project}/${repo}${path}`;
           </TabPanel>
           <TabPanel>
             <HistoryList
```

Comment:
It is not related to this PR but I found out that the pagination isn't preserved when a page is refreshed or on history back. It might be useful if we provide a unique link for a page when a user wants to share a specific history page or visit several revisions to check changes.

Reply:
That will require some kind of pagination link for the datatable.

Reply:
Agreed.